test/test-applications/integrations/TestApplication.NLogBridge/TestApplication.NLogBridge.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <ItemGroup>
+      <PackageReference Include="NLog" VersionOverride="$(LibraryVersion)" />
+      <PackageReference Include="NLog.Extensions.Logging" VersionOverride="$(LibraryVersion)" />
+      <PackageReference Include="Microsoft.Extensions.Logging"/>
+      <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" />
+      <PackageReference Include="System.Diagnostics.DiagnosticSource" />
```

Comment:
I am not fully familiar with NLog details. Is it necessary to bring all these dependencies to make it working? Can this list be reduced?

Reply:
Addressed in [799ef71](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/4371/commits/799ef710f2799637667a61031f2dc6d48b4e149e)